src/components/projectRegister/StepTwoForm.tsx

```
@@ -717,6 +541,18 @@ export default function StepTwoForm({ targetAmount = "", initialData }: StepTwoF
         <h2 className="text-xl font-bold mb-4">상품 상세 설명</h2>
         <MarkdownEditor value={markdown} onChange={setMarkdown} />
       </div>
+
+      {/* 일정 내용 다이얼로그 */}
+      {scheduleDetailsDialogOpen && currentScheduleId !== null && (
+        <ScheduleDetailsDialog
+          isOpen={scheduleDetailsDialogOpen}
+          onClose={() => setScheduleDetailsDialogOpen(false)}
+          details={schedules.find((schedule) => schedule.id === currentScheduleId)?.details || ""}
+          onSave={saveScheduleDetails}
+          scheduleIndex={schedules.findIndex((schedule) => schedule.id === currentScheduleId)}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)